prow/README.md

```diff
@@ -46,12 +46,22 @@ Test with:
 bazel test --features=race //prow/...
 ```
 
+### How to test specific events
+Besides running the default test suite, it is sometimes useful to test prow's
+live behavior. This can be accomplished with `phony`.
```

Comment:
Please note that this is specifically for testing the `hook` component and plugins.